inventory/routes/concepts/inter_warehouse.rst

```diff
@@ -1,131 +1,128 @@
 =====================================
-How to do inter-warehouses transfers?
+Proceed to inter-warehouses transfers
```

Comment:
Honestly, I kept the warehouses because it was the name in the previous version.
But I agree that going straight to the point is far better!

Let's go for "Proceed to". And, when the next display is available, I'll change it again if it's better without the "proceed to" :)

Reply:
Well, @Lau-Odoo that's it.
We do have the new doc now hahaha